docs/systemd-with-workers/workers/federation_sender.yaml

```diff
@@ -0,0 +1,8 @@
+worker_app: synapse.app.federation_sender
+worker_name: federation_sender1
+
+# The replication listener on the main synapse process.
+worker_replication_host: 127.0.0.1
+worker_replication_http_port: 9093
+
+worker_log_config: /etc/matrix-synapse/federation-sender-log.yaml
```

Comment:
In my environment, this configuration is running.

In #12737 was added a config with an undocumented `replication` listener.
https://github.com/matrix-org/synapse/blob/7b7478e8b65cceb9e7362c6c1cb932b569a6f383/contrib/docker_compose_workers/workers/synapse-federation-sender-1.yaml#L1-L12